integration/integration_test_utils.py

```diff
@@ -294,3 +324,32 @@ def _convert_nginx_to_python_type(typ, value):
         return datetime.utcfromtimestamp(secs + nanosecs / 1000000000)
     else:
         raise V3ioError(f"Type {typ} in get item response is not supported")
+
+
+def create_sql_table(schema, table_name, sql_db_path, key):
+    import datetime
+
+    import pandas as pd
+    import sqlalchemy as db
+
+    engine = db.create_engine(sql_db_path, echo=True)
+    with engine.connect() as _:
```

Comment:
Looks to me like this line is completely unnecessary.
```suggestion
```

Reply:
@gtopper I need to connect, to get the updated metadata of our db

Reply:
But if that's the case, why in some other places we call `db.MetaData()` without first calling this? E.g. in `SQLSource._run_loop()`.

Reply:
I entered it inside also, it have to be there because as I said and understood, we have to connect to be able get the updated metadata